src/Form/atoms/Label.tsx

```diff
@@ -16,16 +16,18 @@ const LabelText = styled.span`
 `;
 
 interface OwnProps {
-  htmlFor: string
+  htmlFor?: string
   labelText: string
 }
 type Props = OwnProps & LabelHTMLAttributes<HTMLLabelElement>
 
 const Label : React.FC<Props> = ({ htmlFor, labelText, children, ...props }) => {
-  return <StyledLabel htmlFor={htmlFor} {...props}>
-    <LabelText>{labelText}</LabelText>
-    {children}
-  </StyledLabel>;
+  return (
+    <StyledLabel htmlFor={htmlFor} {...props}>
+      <LabelText>{labelText}</LabelText>
+      {children}
+    </StyledLabel>
+  );
```

Comment:
What's the relevance of adding the brackets to the return here?

Reply:
eslint complains about tags not being aligned.

you could say cosmetic/readability improvement 🤔

Reply:
I've not seen that error before. Perhaps my linter isn't setup properly... I have noticed some of my code being fixed recently for that reason so I suspect I may need to check in to that. Thanks!